yarn-project/end-to-end/src/e2e_token_contract.test.ts

```diff
@@ -489,12 +485,56 @@ describe('e2e_token_contract', () => {
           expect(amount).toBeGreaterThan(0n);
           const nonce = Fr.random();
 
+          const action = asset
+            .withWallet(wallets[1])
+            .methods.transfer_public(accounts[0].address, accounts[1].address, amount, nonce);
+
+          await wallets[0].setPublicAuthWit({ caller: accounts[1].address, action }, true).send().wait();
+
+          await wallets[0].cancelAuthWit({ caller: accounts[1].address, action }).send().wait();
+
+          // Check that the message hash is no longer valid. Need to try to send since nullifiers are handled by sequencer.
```

Comment:
```suggestion
          // Check that the authwit is no longer valid. Need to try to send since nullifiers are handled by sequencer.
```

I am being autistic here but think the use of authwit is clearer here in the comment.

Reply:
An authwit is some authentication witness for a specific message hash. So the thing you as an user is interested in, is whether the message is valid (with a signature OR if in public as here if the message hash directly is approved in storage).